### PR TITLE
4.0.5 snapshot

### PR DESCRIPTION
**Pull request by bpangburn**
I want to do a point release that picks up the latest JdbcRowSetImpl along with some other POM plugin and dependency updates. Then I'll try to pick back up on the formatted text field classes.

Note you'll also see some formatted text field code that was added and then removed because I accidentally committed to the wrong branch.

### CODE REVIEW

File: swingset-demo/pom.xml [hunk 1]
```diff
@@ -22,11 +22,11 @@
 		<version.java>1.8</version.java>
 
 		<version.log4j>2.14.1</version.log4j>
-		<version.swingset>4.0.4</version.swingset>
+		<version.swingset>4.0.5-SNAPSHOT</version.swingset>
```
Comment (errael) [inline swingset-demo/pom.xml:25]: After doing (from parent directory) cd swingset; mvn clean; mvn install. I got an error in swingset-demo. I couldn't find 
`com.nqadmin.swingset:swingset:jar:4.0.5-SNAPSHOT`

Got rid of this "-SNAPSHOT" and things went OK.

**Review by errael — CHANGES_REQUESTED** [review]
With the change so that demo doesn't use the snapshot, built and ran demo with-dependencies. Brought up a few screens; didn't try anything else.

I notice that the enforcer only allows specific versions, and not version X and later.

File: swingset-demo/pom.xml (hunk 1 above)
Comment (bpangburn) [reply swingset-demo/pom.xml]: Removed -SNAPSHOT as suggested.

**Review by errael — APPROVED** [review]
I'm mostly assuming all the newer versions are cool. Wiped 4.0.5* from local repository, updated local clone, built, ran.

**Pull request merged**